### PR TITLE
DEV-3690: Update transactional emails to use header logo alt text

### PR DESCRIPTION
**Pull request by GuiMend**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
n/a

#### What's this PR do?
Adds logo alt text to email templates

#### Why are we doing this? How does it help us?
Improve email templates

#### How should this be manually tested? Please include detailed step-by-step instructions.
- Log in with HUB admin
- Go to style edit modals
- Send test emails
- Check in your inbox the received email
- Inspect browser, focus on header logo
- If style is default style, logo should have alt text = News Revenue Hub
     - Can use "Chicago" style
- If style is custom, logo should either have empty alt text "" if alt-text is not set or it should have the alt text set in default-donation-page style
     - Can use "Style to be available for berk" for custom logo with empty alt
     - Can use "BILLY_PENN" for custom logo with custom alt text (alt="Billy Penn Logo")

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?
no

#### Have automated unit tests been added? If not, why?
yes

#### How should this change be communicated to end users?
n/a

#### Are there any smells or added technical debt to note?
no

#### Has this been documented? If so, where?
no

#### What are the relevant tickets? Add a link to any relevant ones.
https://news-revenue-hub.atlassian.net/browse/DEV-3690

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
no

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:
Parent ticket: https://news-revenue-hub.atlassian.net/browse/DEV-3056

### CODE REVIEW

File: apps/organizations/models.py [hunk 1]
```diff
@@ -293,6 +293,7 @@ class TransactionalEmailStyle:
 
     is_default_logo: bool = False
     logo_url: str = None
+    logo_alt_text: str = None
```
Comment (benjaminEwhite) [inline apps/organizations/models.py:296]: Not a big deal, but any reason can't default to empty string?

File: apps/organizations/models.py [hunk 2]
```diff
@@ -983,6 +985,9 @@ def transactional_email_style(self) -> TransactionalEmailStyle:
             return TransactionalEmailStyle(
                 is_default_logo=not page.header_logo,
                 logo_url=page.header_logo.url if page.header_logo else HubDefaultEmailStyle.logo_url,
+                logo_alt_text=page.header_logo_alt_text or ""
+                if page.header_logo
+                else HubDefaultEmailStyle.logo_alt_text,
```
Comment (benjaminEwhite) [inline apps/organizations/models.py:990]: There's enough complexity wrapped up here that might make sense to bust out into a separate method on page model (or better yet, as a property `DonationPage.logo_alt_text` or whichever name is most appropriate).

**Review by benjaminEwhite — CHANGES_REQUESTED** [review]
@GuiMend looking good overall. A couple of small inline comments, and one other request. We're trying to be more detailed with QA instructions.  Would be helpful to provide links to specific styles / pages that are set up with default vs. non default styles so that part of QA requires less guesswork/setup.

File: apps/organizations/models.py (hunk 1 above)
Comment (GuiMend) [reply apps/organizations/models.py:296]: We can. Updated

File: apps/organizations/models.py (hunk 2 above)
Comment (GuiMend) [reply apps/organizations/models.py:990]: I've made a suggestion. Let me know if it's good enough this way.

**Review by benjaminEwhite — CHANGES_REQUESTED** [review]
@GuiMend I looked more closely, and think we can simplify the logo alt text value assignment.

File: apps/organizations/models.py (hunk 2 above)
Comment (benjaminEwhite) [reply apps/organizations/models.py]: I'm okay with what's here.

File: apps/organizations/models.py [hunk 3]
```diff
@@ -980,9 +982,12 @@ def transactional_email_style(self) -> TransactionalEmailStyle:
             return HubDefaultEmailStyle
         else:
             _style = AttrDict(page.styles.styles if page.styles else {})
+            custom_logo_alt_text = page.header_logo_alt_text or ""
```
[the remark this answers is not in the document]
Comment (benjaminEwhite) [reply apps/organizations/models.py]: This is all good.

File: apps/organizations/models.py (hunk 1 above)
Comment (benjaminEwhite) [reply apps/organizations/models.py]: @GuiMend looks like this small change didn't make it  here (defaulting to empty string).

**Review by benjaminEwhite — APPROVED** [review]

**Pull request merged**